modules/serverless-creating-a-kafka-event-sink.adoc

```diff
@@ -0,0 +1,32 @@
+// Module included in the following assemblies:
+//
+// * serverless/develop/serverless-event-sinks.adoc
+
+:_content-type: PROCEDURE
+[id="serverless-creating-a-Kafka-event-sink_{context}"]
+
+= Creating a Kafka event sink
+
+As a developer, you can create an event sink to receive events from a particular source and send them to a kafka topic. 
```

Comment:
Here I think Kafka with a capital K is needed:

"send them to a Kafka topic"

Reply:
+1 the mention in prereq is with a capital K